controller/token_storage_blackbox_test.go

```diff
@@ -93,7 +93,7 @@ func (rest *TestTokenStorageREST) checkRetrieveOSOServiceAccountToken(saName str
 	service, controller := rest.SecuredControllerWithServiceAccount(sa)
 	require.True(rest.T(), len(rest.Configuration.GetOSOClusters()) > 0)
 	for _, cluster := range rest.Configuration.GetOSOClusters() {
-		_, tokenResponse := test.RetrieveTokenOK(rest.T(), service.Context, service, controller, cluster.URL)
+		_, tokenResponse := test.RetrieveTokenOK(rest.T(), service.Context, service, controller, cluster.URL, nil)
```

Comment:
Add some tests when the force param is not nil and handled properly.

Reply:
Added 2 tests:
06656cecf2653d2e3d7fdf6d064543083f9a0323

1. Token present in DB, but error during interaction with github.com
2. Token present in DB, & success during interaction with github.com